### ISSUE TITLE
[BUG] Notifications Filter button focus is on entire screen instead of the button

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5741]
**Describe the bug**
A filter button (or any interactive component) that moves focus to the entire screen or an unexpected container, rather than highlighting the button itself, violates several Web Content Accessibility Guidelines (WCAG) related to focus management and visibility. 

[WCAG 2.4.7 Focus Visible (Level AA)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html): This requires that any keyboard-operable user interface has a mode of operation where the keyboard focus indicator is visible. If the focus outline is not present on the button but moves to a parent container or disappears, it violates this, as users cannot easily see which element is active.

[WCAG 2.4.3 Focus Order (Level A)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html): If the focus jumps unpredictably around the page or to the container instead of staying on the button, this violates the requirement that navigation order preserves the meaning of the interface.

**To Reproduce**
1. Go to 'Notifications..'
2. Gesture to 'Filter.'

**Expected behavior**
The focus state should be applied to the button it pertains to not the entire screen. 

**Screenshots**

![Image](https://github.com/user-attachments/assets/2be21227-4d5b-47ca-b674-493a7e0b413f)

**Smartphone (please complete the following information):**
 - Device: Google Pixel 8a
 - OS:  Android 16
 - Version 7.3.50

![Image](https://github.com/user-attachments/assets/7e5258ce-8809-42b1-9b78-0b0f14399db6)

 - Device: iPhone 14+
 - OS:  18.6.2
 - Version 7.3.48

![Image](https://github.com/user-attachments/assets/4deb3837-2df4-4e69-b0b6-25c29a06689f)

**Additional context**
To avoid duplication of efforts, during the regression testing of 7.3.50/7.3.48 respectively, other examples of this issue will be added to this ticket.

**Comment by acovher2** [comment]
Another instance of this is on the Customize screen. There is no visual focus on the Customize header, just on the entire modal. Please make the focus visibility indicator on the word Customize only not the entire screen for Android, on iOS it is just not showing at all. 

<img width="442" height="966" alt="Image" src="https://github.com/user-attachments/assets/2fa295e3-799a-4db3-a7a7-79b21fe34b30" />

<img width="426" height="926" alt="Image" src="https://github.com/user-attachments/assets/3d143973-8103-4cc2-afb3-e2a7c1cfa8d3" />

**Comment by TodorBachvarov** [comment]
@acovher2 these issues should be fixed in version 7.3.52 / 7.8.6

**Comment by acovher2** [comment]
Verified as fixed in 8.0.8
